packages/components/src/jetpack-mobile-promo-card/index.tsx

```diff
@@ -1,64 +1,118 @@
+import { WordPressJetpackSVG } from './svg-icons';
 
 export type JetpackMobilePromoCardProps = {
 	className?: string;
-	isMobile?: boolean;
+	isWoo?: boolean;
 };
 
 export default function JetpackMobilePromoCard( {
 	className,
-	isMobile,
+	isWoo,
 }: JetpackMobilePromoCardProps ) {
 	// Using useTranslate() with interpolation to set up the title/message.
 	// https://wpcalypso.wordpress.com/devdocs/packages/i18n-calypso/README.md
 	const translate = useTranslate();
-	const redirectLink = 'https://jetpack.com/app/';
+	const redirectLink = isWoo ? 'https://woo.com/mobile/' : 'https://jetpack.com/app/';
+	const linkClassName = isWoo ? 'woo' : 'jetpack';
 	const components = {
-		a: <a href={ redirectLink } />,
+		a: <a className={ linkClassName } href={ redirectLink } />,
 	};
-	// TODO: Fix mobile logic.
-	// Should this maybe come via a media query in the CSS?
-	// Also, does it make sense to push one app store over the other?
-	// Might make more sense to have a custom "app store" SVG image that goes to wp.com/app.
-
-	// TODO: Include QR code in bundle?
-	// Pulling the QR code from the jetpack.com site for now.
+	const userAgent = window.navigator.userAgent.toLowerCase();
+	const isApple = userAgent.includes( 'iphone' ) || userAgent.includes( 'ipad' );
+	const isGoogle = userAgent.includes( 'android' );
+	let appStoreLink = '#';
+	if ( isApple ) {
+		if ( isWoo ) {
+			appStoreLink = 'https://apps.apple.com/ca/app/woocommerce/id1389130815';
+		} else {
+			appStoreLink = 'https://apps.apple.com/ca/app/jetpack-website-builder/id1565481562';
+		}
+	} else if ( isGoogle ) {
+		if ( isWoo ) {
+			appStoreLink = 'https://play.google.com/store/apps/details?id=com.woocommerce.android';
+		} else {
+			appStoreLink = 'https://play.google.com/store/apps/details?id=com.jetpack.android';
+		}
+	}
 	return (
 		<div className={ classNames( 'promo-card', className ?? null ) }>
 			<div className="promo-lhs">
 				<div className="promo-card__icons">
-					<WordPressJetpackSVG />
+					{ isWoo && <img src={ iconWoo } alt="Icon for the Woo mobile app" /> }
+					{ ! isWoo && <WordPressJetpackSVG /> }
 				</div>
 				<p className="promo-card__title">
-					{ translate( 'Bring your stats with you using the {{a}}Jetpack{{/a}} mobile app', {
-						components,
-					} ) }
+					{ isWoo &&
+						translate( 'Bring your Store stats with you using the {{a}}Woo{{/a}} mobile app', {
+							components,
+						} ) }
+					{ ! isWoo &&
+						translate( 'Bring your stats with you using the {{a}}Jetpack{{/a}} mobile app', {
+							components,
+						} ) }
 				</p>
 				<p className="promo-card__message">
-					{ isMobile
-						? translate(
-								'Check your stats on-the-go and get real-time notifications with the Jetpack mobile app.'
-						  )
-						: translate(
-								'Visit {{a}}jetpack.com/app{{/a}} or scan the QR code to download the Jetpack mobile app.',
-								{ components }
-						  ) }
+					{ isApple &&
```

Comment:
Before composing the returned HTML, could we determine the rendered content with the complex conditions? It would make the logic more explicit outside the HTML structure.